src/services/itemLogin/index.ts

```diff
@@ -14,9 +16,10 @@ import {
 import { ItemLoginService } from './service';
 
 const plugin: FastifyPluginAsync = async (fastify) => {
-  const { db, items } = fastify;
+  const { db } = fastify;
 
-  const iLService = new ItemLoginService(fastify, items.service);
+  const itemService = resolveDependency(ItemService);
+  const iLService = new ItemLoginService(fastify, itemService);
```

Comment:
I think we can rename the constant into `itemLoginService`


```suggestion
  const itemLoginService = new ItemLoginService(fastify, itemService);
```